packages/cli/src/routeGeneration/templates/express.hbs

```diff
@@ -12,7 +12,7 @@ import { expressAuthentication } from '{{authenticationModule}}';
 {{/if}}
 {{#if iocModule}}
 import { iocContainer } from '{{iocModule}}';
-import type { IocContainer, IocContainerFactory } from '@tsoa/runtime';
+import type { IocContainer, IocContainerFactory, RegisterRoutesOptions } from '@tsoa/runtime';
```

Comment:
`RegisterRoutesOptions` should import @ line 4:
```
import { Controller, ValidationService, FieldErrors, ValidateError, TsoaRoute, HttpStatusCodeLiteral, TsoaResponse, fetchMiddlewares, RegisterRoutesOptions } from '@tsoa/runtime';
```

Since Line15 only imports when iocModule is enable.

Reply:
thank you for the heads up...

Reply:
fixed the import as you suggested